main.py

```diff
@@ -31,11 +38,194 @@ def setup(self):
+            for table in pipeline["multi_table"]:
+                table_id = table["table_id"]
+                for napp in table["napps_table_groups"]:
+                    if napp not in content:
+                        content[napp] = {}
+                    for flow_type in table["napps_table_groups"][napp]:
+                        content[napp][flow_type] = table_id
+        return content
+
+    def emit_event(self, name: str, content: dict = None):
+        """Send event"""
+        context = "kytos/of_multi_table"
+        event_name = f"{context}.{name}"
+        event = KytosEvent(name=event_name, content=content)
+        self.controller.buffers.app.put(event)
+
+    @listen_to("kytos/(mef_eline|coloring|of_lldp).enable_table")
+    def on_enable_table(self, event):
+        """Listen for NApps responses"""
+        self.handle_enable_table(event)
+
+    def handle_enable_table(self, event):
+        """Handle NApps responses from enable_table
+        Second, wait for all the napps to respond"""
+        napp = event.name.split('/')[1].split('.')[0]
+        # Check against the last current table
+        self.required_napps.remove(napp)
+        if self.required_napps:
+            # There are more required napps, 'waiting' responses
+            return
+        self.get_flows_to_be_installed()
+
+    def get_flows_to_be_installed(self):
+        """Get flows from flow manager so this NApp can modify them
+        Third, install the flows with different table_id"""
+        pipeline = self.get_enabled_table()
+        if pipeline.get("status") == "enabled":
+            # Pipeline enabling has finnished already
+            return
+        # Keys necessary to delete a flow, 'cookie_mask' is added later
+        keys = {'idle_timeout', 'hard_timeout', 'cookie', 'priority',
+                'match', 'actions'}
+        command = "v2/stored_flows?state=installed"
+        response = requests.get(f"{FLOW_MANAGER_URL}/{command}")
+
+        if response.status_code // 100 != 2:
+            log.error(f"Could not get the flows from flow_mager. Status "
+                      f"code {response.status_code}.")
+            # Problem here: pipeline stays as 'enabling', error needs
+            # to be handled. Events have been sent and NApps are probably
+            # working in a different pipeline setup
+            return
+        if pipeline.get("status") is None:
+            # Changing to default pipeline. Miss flow entries are not needed
+            self.delete_miss_flows()
+        flows_by_swich = response.json()
+        set_up = self.build_content(pipeline)
+
+        for switch in flows_by_swich:
+            delete_flows = {"flows": []}
+            install_flows = {"flows": []}
+            for flow in flows_by_swich[switch]:
+                owner = flow["flow"]["owner"]
+                if owner not in set_up:
+                    continue
+                expected_table_id = set_up[owner][flow["flow"]["table_group"]]
```

Comment:
Just a heads up that changes that I request on `flow_manager` will make the `flow["flow"]["table_group"]` to potentially trow a `KeyError`, can you also check if the value is truthy to be safe?